app/src/main/java/com/teampophory/pophory/feature/album/detail/AlbumDetailViewModel.kt

```diff
@@ -11,41 +12,23 @@ import javax.inject.Inject
 
 @HiltViewModel
 class AlbumDetailViewModel @Inject constructor(
-    private val photoRepository: PhotoRepository
+    private val photoRepository: PhotoRepository,
+    private val savedStateHandle: SavedStateHandle
 ) : ViewModel() {
-    private val _id = MutableLiveData<Int>()
-    val id: LiveData<Int> get() = _id
 
-    private val _studio = MutableLiveData<String>()
-    val studio: LiveData<String> get() = _studio
+    private val _photoDetailInfo = MutableLiveData<PhotoDetailInfo>()
+    val photoDetailInfo: LiveData<PhotoDetailInfo> get() = _photoDetailInfo
```

Comment:
```suggestion
    private val _photoDetail = MutableLiveData<PhotoDetail>()
    val photoDetail: LiveData<PhotoDetail> get() = _photoDetail
```

Detail, Info 모두 비슷한 의미 지니고 있어서 하나만 사용해도 될듯. 근데 Info는 너무 추상적인 의미여서(Data랑 거의 같은 범주) 웬만하면 사용을 지양하면 좋을 것 같음

Reply:
사용 지양하기 메모...📝